site.yml

```diff
@@ -28,8 +28,8 @@
     - docker
     - weave
     - dnsmasq
-    - registrator
     - cadvisor
+    - registrator
```

Comment:
just wondering the significance of the movement here? Wondering if that will affect what i've done over here https://github.com/Capgemini/Apollo/pull/504/files (where we get the cadvisor IP) - i'm guessing we might need to refactor that PR anyway based on this PR (or vice versa)